src/test/java/spimdata/OmeZarrWithWriterTest.java

```diff
@@ -103,4 +183,31 @@ void writeAndReadImageBdvOmeZarr() throws SpimDataException {
 
         new SpimDataOpener().openSpimData( xmlPath, format );
     }
+
+    @Test
+    void checkOmeZarrEdges() throws SpimDataException {
```

Comment:
@K-Meech Are you specifically checking the edges only? If not, maybe change the name of the method?

Reply:
No - I'm checking the whole of the downsampled image. I changed the name to ```checkOmeZarrLoopBack```, as it's catching errors in the 'loopBack' to previously written levels.